rules/manager_test.go

```diff
@@ -784,42 +784,35 @@ func TestUpdate(t *testing.T) {
 
 // ruleGroupsTest for running tests over rules.
 type ruleGroupsTest struct {
-	Groups []ruleGroupTest `yaml:"groups"`
+	Groups []rulefmt.RuleGroup `yaml:"groups"`
 }
 
 // ruleGroupTest forms a testing struct for running tests over rules.
-type ruleGroupTest struct {
-	Name     yaml.Node      `yaml:"name"`
-	Interval model.Duration `yaml:"interval,omitempty"`
-	Rules    []rule         `yaml:"rules"`
-}
-
-type rule struct {
-	Record      string            `yaml:"record,omitempty"`
-	Alert       string            `yaml:"alert,omitempty"`
-	Expr        string            `yaml:"expr"`
-	For         model.Duration    `yaml:"for,omitempty"`
-	Labels      map[string]string `yaml:"labels,omitempty"`
-	Annotations map[string]string `yaml:"annotations,omitempty"`
-}
+//type ruleGroupTest struct {
```

Comment:
This is left over

Reply:
Ahh, will remove in the following commit soon. BTW, can you say why is the `test` failing?

Reply:
No idea offhand.

Reply:
Got it, it was at `t.Fail()`. Now, fixed it.